right_hand_offset:
- 0.08176150241186875
- 0.042509600848533724
- -0.01
